### PR TITLE
Movie seat booking

### PR DESCRIPTION
**Pull request by HannaSyn**
[Demo](https://hannasyn.github.io/movie_seats/)

Please, review!

### CODE REVIEW

File: submissions/hannasyn/movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+          <h1 class="movie__title">Soul</h1>
+          <img class="movie__poster" src="assets/soul.jpg" alt="soul_poster"
+          srcset="
+          assets/soul_mobile.jpg 126w,
+          assets/soul_1440.jpg 260w,
+          assets/soul_4k.jpg 401w"
+          sizes="(max-width: 500px) 126px,
+            (max-width: 1440px) 260px,
+            400px"
+          >
```
Comment (A-Ostrovnyy) [inline submissions/hannasyn/movie-seat-booking/index.html:35]: 🔥

File: submissions/hannasyn/movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+        </div>
+      </article>
+      <section class="hall">
+        <h2 class="hall__title">Pick your seat</h2>
+        <p class="hall__screen">Screen</p>
+        <form action="#" class="seats">
+          <div class="seats__row">
+            <div class="seats__column">
+              <label class="seats__item">
+                <input class="seats__input input-hidden" type="checkbox" name="seat" value="1">
```
Comment (A-Ostrovnyy) [inline submissions/hannasyn/movie-seat-booking/index.html]: If I check this seat and seat from line 155, how you understand the difference between them with the same value?)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comment below!

File: submissions/hannasyn/movie-seat-booking/index.html (hunk 2 above)
Comment (HannaSyn) [reply submissions/hannasyn/movie-seat-booking/index.html]: Thank you for pointing! My bad, I fixed it)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**